src/mirror-client-finder/directory-finder.ts

```diff
@@ -20,6 +21,11 @@ export async function findClientDir(client: InstallableClient, desiredVersionRan
     );
     const clientVersionedDir = clientBaseDir + clientMatchedVersion;
 
+    if (client === Inputs.CRC && getOS() === "macos" && semver.gt(clientMatchedVersion, "1.27.0")) {
```

Comment:
```suggestion
    if (client === Inputs.CRC && getOS() === "macos" && semver.gte(clientMatchedVersion, "1.28.0")) {
```
this way if there is a 1.27.1 release that still has the .tar.xz it will work